### ISSUE TITLE
[5.x] non minified css

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#43241]
Is it intentional that we only ship the non-minified css for
awesomplete
jquery-minicolors
chosen

or is it a bug in our build scripts?

**Comment by dgrammatiko** [comment]
> Is it intentional that we only ship the non-minified 

yes, these packages do not provide non minified versions, thus the npm build script has exceptions for those

**Comment by brianteeman** [comment]
so why dont we minify them?

**Comment by dgrammatiko** [comment]
Actually I was wrong here, the part that I was referring is about JS: https://github.com/joomla/joomla-cms/blob/c594155371fc5c8efdf3039d4de6a80fc09068c3/build/build-modules-js/init/minify-vendor.es6.js#L8

About the css you're asking:
- awesomplete has a .min.css file that is not really minified
- jquery minicolors has a non minified file jquery.minicolors.css
- chosen is abandoned and should have been removed in 5.x

Although you're right and these files should've both a minified and a non minified version this needs an exception in the tooling. IMO all these packages could/should be dropped instead of patching them here...

**Comment by brianteeman** [comment]
keeping them or dropping them is a different discussion. There might be other vendor files that only ship non-minified css in the future

while they are present they should be minified and the tooling needs to be updated to do this.

**Comment by richard67** [comment]
The debugbar css `media/vendor/debugbar/debugbar.css` is not minified, too, and also not gzipped when building a package on the current 5.23-dev branch. In 5.2 beta 2 we had it minified and gzipped.

**Comment by dgrammatiko** [comment]
The gzip works only on `.min.[css|js]` all the files from debugbar are `.css` or `.js` so are skipped

**Comment by richard67** [comment]
> The gzip works only on `.min.[css|js]` all the files from debugbar are `.css` or `.js` so are skipped

But in 5.2 beta 2 as well as in 4.4 we have a minified file `media/vendor/debugbar/debugbar.min.css` and a gzipped file `media/vendor/debugbar/debugbar.min.css.gz`.

So it seems something has changed between 5.2 beta 2 and now, and I would call that a regression.

**Comment by dgrammatiko** [comment]
Oh, is that due to changes on the tooling or upstream?

**Comment by richard67** [comment]
Good question. I have no idea.

**Comment by richard67** [comment]
Got it: The debugbar css is broken, so npm ci shows an error and doesn't minify it. See https://github.com/maximebf/php-debugbar/releases/tag/v1.23.0 and https://github.com/maximebf/php-debugbar/pull/675 . We just need to update to that version. PR is on the way (but not by me).

**Comment by brianteeman** [comment]
pr done
